src/frontend/planner_test/tests/testdata/output/sink_into_table.yaml

```diff
@@ -0,0 +1,14 @@
+# This file is automatically generated. See `src/frontend/planner_test/README.md` for more information.
+- id: create sink into table having default expression with now()
```

Comment:
Do we have existing tests for default values in `SINK INTO TABLE` besides `NOW()`?

Reply:
In e2e_test/sink/sink_into_table we test default values and generated columns. But we do not have many planner test, I will take a look to add some.